MANIFEST.in

```diff
@@ -7,5 +7,4 @@ include docs/conf.py
 include docs/_static/*
 include fabfile.py
 include tox.ini
-include more_itertools/py.typed
```

Comment:
Hmm, looks like some merge issues caused this to be added to the diff. Could you remove it?